templates/datadog.yaml.j2

```diff
@@ -1,7 +1,14 @@
 # Managed by Ansible
 
-{% if datadog_config["dd_url"] is not defined -%}
-dd_url: {{ datadog_url | default('https://app.datadoghq.com') }}
+{% if datadog_config["dd_url"] is not defined
```

Comment:
I don't think we need to handle this logic here, we can let the Agent handle the options' precedence